cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,34 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Results Visibility') %></h3>
```

Comment:
Per my comment earlier, I'd suggest changing this to "Assessment Results Visibility" and instances of "problem"/"problems" to "assessment"/"assessments" for consistency.